src/main/scala/it/pagopa/interop/apigateway/api/impl/GatewayApiServiceImpl.scala

```diff
@@ -85,7 +87,7 @@ final case class GatewayApiServiceImpl(
     contexts: Seq[(String, String)],
     toEntityMarshallerProblem: ToEntityMarshaller[Problem]
   ): Route = authorize {
-    logger.info(s"Upserting tenant with extenalId (${origin},${externalId}) and attribute $code")
```

Comment:
😠

Reply:
you should stop to add curly braces in this case, please 🤣

Reply:
no! and I will enforce it with the formatted 😈